fitlins/interfaces/bids.py

```diff
@@ -232,9 +232,8 @@ def _load_level1(self, runtime, analysis):
                 # t/aCompCor
                 # We may want to add criteria to include HPF columns that are not
                 # explicitly listed in the model
-                names = [col for col in dense.columns
-                         if col.startswith('non_steady_state') or
-                         col in step.model['x']]
+                names = step.model['x'].copy()
```

Comment:
How's this? (I haven't tried it. I assume Pandas' columns object supports `in`.)

```suggestion
                names = [var for var in step.model['x'] if var in dense.columns]
```

Reply:
Should be OK in theory